exchanges/request/request_types.go

```diff
@@ -3,73 +3,52 @@ package request
 import (
 	"io"
 	"net/http"
-	"sync"
 	"time"
 
 	"github.com/thrasher-corp/gocryptotrader/common/timedmutex"
 	"github.com/thrasher-corp/gocryptotrader/exchanges/nonce"
 )
 
-var supportedMethods = []string{http.MethodGet, http.MethodPost, http.MethodHead,
-	http.MethodPut, http.MethodDelete, http.MethodOptions, http.MethodConnect}
-
 // Const vars for rate limiter
 const (
-	DefaultMaxRequestJobs       = 50
-	DefaultTimeoutRetryAttempts = 3
-	DefaultMutexLockTimeout     = 50 * time.Millisecond
-	proxyTLSTimeout             = 15 * time.Second
+	DefaultMaxRequestJobs       int32 = 50
```

Comment:
are we going to need 2147483647 max jobs?
uint8 will give up to 255

Reply:
Yes.

Reply:
Yes.